### PR TITLE
Feature/transaction type

### PR DESCRIPTION
**Pull request by Alphonce-Mutebi**

### CODE REVIEW

File: src/components/buttons/KeyPad.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  key: {
+    width: keyWidth,
+    height: keyHeight,
+    borderRadius: keyWidth / 2,
+    justifyContent: "center",
+  },
+
+  number: {
+    fontSize: 24,
+    lineHeight: 26,
```
Comment (Allen-Muhani) [inline src/components/buttons/KeyPad.tsx:162]: Kindly pick the colors from the FONT file.

File: src/components/cards/AcceptTXRequestDetailsCard.tsx [hunk 2]
```diff
@@ -3,10 +3,9 @@ import React from "react";
 import { Text, View, StyleSheet } from "react-native";
 import { TextInputMask } from "react-native-masked-text";
 
-import { FONTS } from "../../font/fonts/fonts";
+import { FONTS } from "../../styles/fonts/fonts";
```
Comment (Allen-Muhani) [inline src/components/cards/AcceptTXRequestDetailsCard.tsx:6]: What's the difference between the two?

File: src/components/menus/BottomMenu.tsx [hunk 3]
```diff
@@ -5,18 +5,32 @@ import { Pressable } from "react-native";
 import { COLORS } from "../../styles/theme";
 import SendButton from "../buttons/SendButton";
 import WithdrawButton from "../buttons/WithdrawButton";
+import { useNavigation, useRoute } from "@react-navigation/native";
 
-const BottomMenu = ({navigation}) => {
+const BottomMenu = ({ navigation }) => {
   return (
-    <View style={{ flexDirection: 'column' }}>
+    <View style={{ flexDirection: "column" }}>
```
Comment (Allen-Muhani) [inline src/components/menus/BottomMenu.tsx:12]: Why the change?

File: src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+  image: {
+    height: 150,
+    maxWidth: SIZES.width * 0.8,
+    resizeMode: "contain",
+    marginBottom: 20,
+  },
+
+  title: {
+    fontSize: 18,
```
Comment (Allen-Muhani) [inline src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx:122]: try ...FONTS.sh2 instead of re-writing the text formatting styles

File: src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+    lineHeight: 22,
+    color: "#2C2948",
+    textAlign: "center",
+    fontFamily: FONTS.sh2.fontFamily,
+  },
+
+  text: {
+    fontSize: 14,
+    lineHeight: 21,
+    color: COLORS.textPrimary,
```
Comment (Allen-Muhani) [inline src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx:132]: try ...FONTS.h1 instead of re-writing the text formatting styles

File: src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+  buttonContainer: {
+    height: 50,
+    width: 100,
+    justifyContent: "center",
+  },
+
+  button: {
+    fontSize: 14,
+    lineHeight: 23,
+    color: "#4840BB",
```
Comment (Allen-Muhani) [inline src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx:146]: try ...FONTS.h5 instead of re-writing the text formatting styles
and pick the color from the COLORs file

File: src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+
+  image: {
+    height: 35,
+    maxWidth: SIZES.width,
+    resizeMode: "contain",
+  },
+
+  title: {
+    fontSize: 18,
+    lineHeight: 22,
```
Comment (Allen-Muhani) [inline src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx:182]: Pick the font from the fonts file

File: src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+    color: "#4840BB",
+    fontFamily: FONTS.body3.fontFamily,
+  },
+
+  subTitle: {
+    fontSize: 16,
+    lineHeight: 28,
```
Comment (Allen-Muhani) [inline src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx:189]: Pick the font from the fonts file

File: src/screens/mpesaToCusd/TransactionType.tsx [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+  title: {
+    fontSize: 28,
+    lineHeight: 34,
+    color: COLORS.primary,
+    fontFamily: FONTS.displayBold.fontFamily,
+  },
+
+  subTitle: {
+    fontSize: 12,
+    lineHeight: 14,
```
Comment (Allen-Muhani) [inline src/screens/mpesaToCusd/TransactionType.tsx:111]: Pick the font from the fonts file

File: src/screens/mpesaToCusd/TransactionType.tsx [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+
+const styles = StyleSheet.create<any>({
+  container: {
+    margin: 30,
+  },
+
+  title: {
+    fontSize: 28,
+    lineHeight: 34,
+    color: COLORS.primary,
```
Comment (Allen-Muhani) [inline src/screens/mpesaToCusd/TransactionType.tsx:105]: Pick the font from the fonts file

File: src/screens/mpesaToCusd/TransactionType.tsx [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+    justifyContent: "center",
+    alignItems: "center",
+    borderRadius: 8,
+    marginTop: 15,
+  },
+
+  buttonTitle: {
+    fontSize: 18,
+    lineHeight: 22,
+    color: COLORS.primary,
```
Comment (Allen-Muhani) [inline src/screens/mpesaToCusd/TransactionType.tsx:152]: Pick the font from the fonts file

File: src/screens/mpesaToCusd/TransactionType.tsx [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+    marginTop: 15,
+    fontFamily: FONTS.body9.fontFamily,
+  },
+
+  buttonSubTitle: {
+    fontSize: 12,
+    lineHeight: 14,
+    color: COLORS.textPrimary,
```
Comment (Allen-Muhani) [inline src/screens/mpesaToCusd/TransactionType.tsx:160]: Pick the font from the fonts file

**Review by Allen-Muhani — CHANGES_REQUESTED** [review]

File: src/components/cards/AcceptTXRequestDetailsCard.tsx (hunk 2 above)
Comment (Alphonce-Mutebi) [reply src/components/cards/AcceptTXRequestDetailsCard.tsx:6]: We are using the one in green based on the folder structure

**Review by steffenkrogmann — APPROVED** [review]
merge after addressing Allen's questions/comments @Alphonce-Mutebi

**Review by Mutinda-Kioko — APPROVED** [review]
